gam_app/views.py

```diff
@@ -328,6 +328,35 @@ def advanced_search_submit(request):
         context['state'] = context['results']
         print(type(context['state']))
         #print(context)
+
+        # to determine the number of returning objects for each model
+        persona_count = 0
+        lugar_count = 0
+        imagen_count = 0
+
+        persona = []
+        lugar = []
+        imagen = []
+
+        for obj in context['results']:
+            if obj.__class__.__name__ == 'Persona':
```

Comment:
Guess this has already shipped but a better way to do this is `isinstance(obj, Persona)`